AppVeyor.psm1

```diff
@@ -53,13 +53,26 @@ function Invoke-AppveyorInstallTask
                               -ChildPath 'nuget.exe'
     Install-NugetExe -OutFile $nugetExePath
 
+    $pesterParameters = @{
+        Name = 'Pester'
+        Force = $true
+    }
+
+    $installModuleSupportsSkipPublisherCheck = (Get-Command Install-Module).Parameters['SkipPublisherCheck']
+    if ($installModuleSupportsSkipPublisherCheck)
+    {
+        $pesterParameters += @{
+            SkipPublisherCheck = $true
+        }
+    }
+
     if ($PesterMaximumVersion)
     {
-        Install-Module -Name Pester -MaximumVersion $PesterMaximumVersion -Force -SkipPublisherCheck
+        Install-Module @pesterParameters -MaximumVersion $PesterMaximumVersion
```

Comment:
Done. I change SkipPublisherCheck parameter to have the same format also, as you suggested for MaximumVersion. 

New commit tested OK here for older build worker image:
https://ci.appveyor.com/project/johlju/xfailovercluster/build/1.6.56.0

New commit tested OK here for Visual Studio 2017 build worker image:
https://ci.appveyor.com/project/johlju/xfailovercluster/build/1.6.57.0